examples/cluster/playground/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:trusty
+FROM ubuntu:bionic
```

Comment:
this could cause to weird differences in behavior between example-cluster and what's running on stages/prod?

Reply:
yeah, can we just use xenial?